Brink Innovation/Example CHT application/disease-surveillance/forms/app/household_member_assessment.properties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "icon": "household-member-assessment",
+  "context": {
+    "place": false,
+    "person": true,
+    "expression": "!contact.date_of_death && !summary.muted && ageInYears(contact) > 0"
```

Comment:
Is this by design please ? Are under 5 also concerned by this form?
 
 BTW I am not able to see the `Under 5 Assessment Form`?